tests/deployments/helm/helm_test.py

```diff
@@ -0,0 +1,112 @@
+    clusterrolebinding_yaml = yaml.load(open(CLUSTERROLEBINDING_YAML_PATH).read())
+    name = clusterrolebinding_yaml.get("metadata", {}).get("name")
+    assert name, "name not found in %s" % CLUSTERROLEBINDING_YAML_PATH
+    print("Creating %s cluster role binding ..." % name)
+    create_clusterrolebinding(body=clusterrolebinding_yaml)
+    print(minikube.exec_kubectl("describe clusterrolebinding %s" % name))
+
+
+def update_values_yaml(minikube, backend, namespace):
+    values_yaml = None
+    with open(os.path.join(LOCAL_CHART_DIR, "values.yaml")) as fd:
+        values_yaml = yaml.load(fd.read())
+        values_yaml["signalFxAccessToken"] = "testing123"
+        values_yaml["clusterName"] = minikube.cluster_name
+        values_yaml["namespace"] = namespace
+        values_yaml["ingestUrl"] = "http://%s:%d" % (backend.ingest_host, backend.ingest_port)
+        values_yaml["apiUrl"] = "http://%s:%d" % (backend.api_host, backend.api_port)
+        values_yaml["monitors"] = yaml.load(MONITORS_CONFIG)
+    with tempfile.NamedTemporaryFile(mode="w") as fd:
+        fd.write(yaml.dump(values_yaml))
+        fd.flush()
+        copy_file_into_container(fd.name, minikube.container, os.path.join(CONTAINER_CHART_DIR, "values.yaml"))
+
+
+def init_helm(minikube):
+    print(minikube.exec_cmd("helm init"))
+    print("Waiting for tiller-deployment to be ready ...")
+    assert wait_for(
+        p(deployment_is_ready, "tiller-deploy", "kube-system"), timeout_seconds=30, interval_seconds=2
+    ), "timed out waiting for tiller-deployment to be ready!"
+
+
+def get_chart_name_version():
+    chart_path = os.path.join(LOCAL_CHART_DIR, "Chart.yaml")
+    chart_name = None
+    chart_version = None
+    with open(chart_path) as fd:
+        chart_yaml = yaml.load(fd.read())
+        chart_name = chart_yaml.get("name")
+        chart_version = chart_yaml.get("version")
+    assert chart_name, "failed to get chart name from %s" % chart_path
+    assert chart_version, "failed to get chart version from %s" % chart_path
+    return chart_name, chart_version
+
+
+def get_daemonset_name(minikube, namespace):
+    chart_name, chart_version = get_chart_name_version()
+    chart_release_name = chart_name + "-" + chart_version
+    output = minikube.exec_cmd("helm list --namespace=%s --output=yaml" % namespace)
+    print(output)
+    release = None
+    for rel in yaml.load(output).get("Releases", []):
+        if rel.get("Chart") == chart_release_name:
+            release = rel
+            break
+    assert release, "chart '%s' not found in helm list output:\n%s" % (chart_release_name, output)
+    release_name = release.get("Name")
+    assert release_name, "failed to get name for release:\n%s" % yaml.dump(release)
+    return release_name + "-" + chart_name
+
+
+def install_helm_chart(minikube, namespace):
+    minikube.exec_cmd("helm install --namespace=%s --dry-run --debug %s" % (namespace, CONTAINER_CHART_DIR))
```

Comment:
What is the point of this dry run?  It doesn't even print the output of it.

Reply:
Will remove.